<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Solution</class>
 <widget class="QMainWindow" name="Solution">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lab_levelName">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>240</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image:ul();
font: 36pt &quot;华文新魏&quot;;
color: rgb(214, 143, 0);</string>
    </property>
    <property name="text">
     <string>横刀立马</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>90</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="lab_present">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/btn.jpg);
color: rgb(255, 85, 0);
font: 20pt &quot;华文新魏&quot;;</string>
       </property>
       <property name="text">
        <string>当前步数</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="num_present">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/btn.jpg);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>660</y>
      <width>411</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btn_lastStep">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/btn.jpg);
color: rgb(255, 85, 0);
font: 22pt &quot;华文新魏&quot;;</string>
       </property>
       <property name="text">
        <string>上一步</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_back">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/btn.jpg);
color: rgb(255, 85, 0);
font: 22pt &quot;华文新魏&quot;;</string>
       </property>
       <property name="text">
        <string>返回</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_nextStep">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/btn.jpg);
color: rgb(255, 85, 0);
font: 22pt &quot;华文新魏&quot;;</string>
       </property>
       <property name="text">
        <string>下一步</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>90</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="lab_best">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/btn.jpg);
color: rgb(255, 85, 0);
font: 20pt &quot;华文新魏&quot;;</string>
       </property>
       <property name="text">
        <string>最优解</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLCDNumber" name="num_best">
       <property name="styleSheet">
        <string notr="true">border-image: url(:/image/btn.jpg);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
